src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -26,13 +33,6 @@
 import org.openmrs.contrib.qaframework.page.RequestAppointmentPage;
 import org.openmrs.contrib.qaframework.page.VisitNotePage;
 
-import io.cucumber.java.After;
-import io.cucumber.java.Before;
```

Comment:
This is going to bring in more noise even almost the whole module, since its not documented anywhere am not still convinced to the changes.

Reply:
isn't it a good idea to align the code convention with other OpenMRS repos convention?

> This is going to bring in more noise even almost the whole module,

isn't this our role as QA Testers to align the module to follow the better convention? for quality assurance?